paymentsheet/src/androidTest/java/com/stripe/android/paymentsheet/utils/TestRules.kt

```diff
@@ -30,16 +28,17 @@ class TestRules private constructor(
         fun create(
             composeTestRule: ComposeTestRule = createEmptyComposeRule(),
             networkRule: NetworkRule = NetworkRule(),
-            analyticEventRule: AnalyticEventRule = AnalyticEventRule()
+            block: (chain: RuleChain) -> RuleChain = { it }
```

Comment:
```suggestion
            block: RuleChain.() -> RuleChain = { this }
```

Would likely make the call sites more idiomatic.

Reply:
Then you could also call block at the end of the chain.